languages/en.php

```diff
@@ -154,6 +154,7 @@
 	'access:write' => "Write access",
 	'access:admin_only' => "Administrators only",
 	'access:missing_name' => "Missing access level name",
+	'access:comments:change' => "There are comments on this content, by changing the access level you will also be changing the access of those comments.  Do you want to continue?",
```

Comment:
If anyone can think of anything better here please chime in.  The number of comments is passed in as a potential variable, I just couldn't think of anything that fit nicely with our grammatical structures...
"There is 1 comment..."
"There are 10 comments..."